lib/ontologies_linked_data/models/mod/semantic_artefact_catalog.rb

```diff
@@ -0,0 +1,228 @@
+require 'yaml'
+
+module LinkedData
+    module Models
+        class SemanticArtefactCatalog < LinkedData::Models::Base
+
+
+            # Load the scheme file once
+            SCHEME_PATH = File.join(__dir__, '../../../../config/schemes/semantic_artefact_catalog.yml')
```

Comment:
there is already a feature like this implemented for the submission model, reuse don't re implement.

Reply:
if you are taking about this
```ruby
model :ontology_submission, scheme: File.join(__dir__, '../../../config/schemes/ontology_submission.yml'),
```
i tried with this one, but we want to bring the default value from the semantic_artefact_catalog.yml file, so i did this to transform the default values from the yaml file to a proc or lambda
why: because in goo when i want to save he only accept the default value that are lambda functions

Reply:
> why: because in goo when i want to save he only accept the default value that are lambda functions
Fix this in Goo to accept strings.